cuda2gcn/src/math.cl

```diff
@@ -279,12 +279,39 @@ ATTR double __nv_modf(double x, __private double *ptr) { return __ocml_modf_f64(
 //-------- T __nv_modff
 ATTR float __nv_modff(float x, __private float *ptr) { return __ocml_modf_f32(x, ptr); }
 
+//-------- T __nv_normcdf
+ATTR double __nv_normcdf(double x) { return __ocml_ncdf_f64(x); }
+
+//-------- T __nv_normcdff
+ATTR float __nv_normcdff(float x) { return __ocml_ncdf_f32(x); }
+
+//-------- T __nv_normcdfinv
+ATTR double __nv_normcdfinv(double x) { return __ocml_ncdfinv_f64(x); }
+
+//-------- T __nv_normcdfinvf
+ATTR float __nv_normcdfinvf(float x) { return __ocml_ncdfinv_f32(x); }
+
+//-------- T __nv_powi
+ATTR double __nv_powi(double x, int n) { return __ocml_pown_f64(x, n); }
+
+//-------- T __nv_powi
+ATTR float __nv_powif(float x, int n) { return __ocml_pown_f32(x, n); }
+
 //-------- T __nv_remquo
 ATTR double __nv_remquo(double x, double y, __private int *ptr) { return __ocml_remquo_f64(x, y, ptr); }
 
 //-------- T __nv_remquof
 ATTR float __nv_remquof(float x, float y, __private int *ptr) { return __ocml_remquo_f32(x, y, ptr); }
 
+//-------- T __nv_saturatef
+ATTR float __nv_saturatef(float x)
+{
+    if (isnan(x)) return 0.0f;
```

Comment:
This function is a simple clamp.  The entire body can be simply written as return __ocml_min_f32(__ocml_max_f32(x, 0.0f), 1.0f);